### DISCUSSION TITLE
New to HAS

### DISCUSSION THREAD

**Discussion by This-is-me-yes** [hawky358/hass_ids_hyyp#9] [General]
Any chance you can share the panel configs as Yawl is not being my friend ... Love this integration been looking for this for months

**Comment by hawky358** [comment]
Hello,

Can you clarify? What do you mean share the panel configs?

**Reply by This-is-me-yes** [reply]
Thanks for the reply ... I wanted to create the same kind of panels you show as the demo ... looking into the YAML configs is making me confused as this is very new to me ... I get the names etc ... but the code baffles me ... I was hoping to create a custom card that looks like the IDS touch keypad ... but would not know where to start ....

**Reply by hawky358** [reply]
I see, sure. It's a bit difficult to know where you are. 
I have made a Discord server for any General discussions around HA and IDS etc. 
I will see if there's any interest, but for now feel free to drop in there. It will be easier to assist with something a bit more "live" than github discussions.

[https://discord.gg/9xVTgHWTZ6](https://discord.gg/9xVTgHWTZ6)
